### ISSUE TITLE
No database select when querying with the Query tab

### ISSUE THREAD

**Issue by jonathangreco** [ragnar-lodbrok/meow-sql#55]
![image](https://user-images.githubusercontent.com/1587180/93492212-da8ec400-f90a-11ea-85dd-dd23fa1baef7.png)
A database is selected (a tab on the top show the DB, and on the left a green tick shows that also.

Once we are on a DB context the query panel sould work on this context.

**Comment by jonathangreco** [comment]
to fix this before our request we need to add the
`USE myDB` command

**Comment by ragnar-lodbrok** [comment]
Sorry, can't reproduce, works for me. 
Do you make extra steps like refresh or you just open session, then db and it doesn't work from the beginning?

**Comment by jonathangreco** [comment]
Yes I can't reproduce either... I will investigate further and com back to you as soon as I get the step to reproduce this.

Thanks !

**Comment by amosbird** [comment]
I just encountered the same issue. It seems the session is expired. I need to reconnect to the database to make it work again.
